crypto/src/signature/ed25519.rs

```diff
@@ -11,7 +11,23 @@ pub type PublicKey = ed25519_dalek::VerifyingKey;
 pub type PrivateKey = ed25519_dalek::SigningKey;
 
 #[cfg(not(feature = "std"))]
-use alloc::{string::ToString as _, vec::Vec};
+use alloc::{format, string::ToString as _, vec::Vec};
+
+fn parse_fixed_size<T, E, F, const SIZE: usize>(payload: &[u8], f: F) -> Result<T, ParseError>
```

Comment:
We have both `f`'s and custom names for `Fn...` in the code. Do you think we should name it `f` here or maybe use some custom name?
Besides that everything looks good.

Reply:
Yeah, naming things is good. Renamed to `fixed_parser` (because it parses a fixed-size array)